graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/QueryFetcherTest.java

```diff
@@ -50,4 +57,32 @@ public void errorTest(String graphQlMutation, String expectedError) {
           "argument 'options.limit' with value 'FloatValue{value=1.0}' is not a valid 'Int'"),
     };
   }
+
+  @ParameterizedTest
+  @MethodSource("operationsWithOptions")
+  @DisplayName("Should execute GraphQL with options passed correctly")
+  public void consistencyAndPagingOptionsTest(
+      String graphQlOperation, Parameters expectedParameters) {
+    ExecutionResult result = executeGraphQl(graphQlOperation);
+    assertThat(result.getErrors()).isEmpty();
+    assertThat(parametersCaptor.getValue()).isEqualTo(expectedParameters);
```

Comment:
Good idea to make this a separate test.

We might want to extract this logic to a parent class like `assertSuccess`, but we can do it later when needed.